oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -70,15 +77,22 @@ public AttestationClient() {}
    *
    * @param url must contain a protocol used for the connection ("https://" or "http://")
    */
-  public void attest(String url, Predicate<ConfigurationInfo> verifier)
+  public void attest(String url, String apiKey, Predicate<ConfigurationInfo> verifier)
```

Comment:
Update doc, also clarify what happens if apiKey is empty / null

Reply:
Done